communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+
+> BinderHubs package together really fantastic tools like notebooks, docker, cloud computing resources, coordination engines, along with clear version control to deliver reproducible research at the click of a button.
+>
+> -- Dr Kirstie Whitaker
+
+![BinderHub](https://zenodo.org/api/iiif/v2/e4125eaf-b456-4097-85fc-6a2e80482d1c:96c70193-2f9e-442d-8cf8-21485d8864e1:1728_TURI_Book%20sprint_45%20repo2docker_040619_v2_MK.jpg/full/750,/0/default.jpg)
+
+## The Binder Federation
+
+[The Binder Federation](https://blog.jupyter.org/the-international-binder-federation-4f6235c1537e) is an experiment in open infrastructure for data science and beyond.
+It is a community of scientists and open-source developers dedicated to scientific reproducibility, communicating ideas through interactive computing, and providing real-time access to results and analyses.
+The Binder Federation provides interactive data analytics environments to roughly 10,000 unique repositories via 300,000 launches a month, and it relies on its team of partner BinderHubs to collaborate in providing this service.
+
+GKE, OVH, GESIS and now new members the Turing pool their resources to create a sustainable, open-source infrastructure that allows easy creation, sharing of and access to reproducible computational environments.
+Pooling resources in such a way is an important step for the resilience and sustanability of Project Binder since the dependency between mybinder.org and its funding is now spread across many sources.
+It also facilitates the decentralisation of the knowledge required to deploy and manage a BinderHub.
+
+_Turing Way_ contributor [Dr Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson) is also a [mybinder.org](https://mybinder.org) operator.
+The operators are responsible for the day-to-day operations and maintainance of the main website.
+She was therefore perfectly positioned to lead this project and bring the Turing into the Binder Federation.
+
+> I'm so excited that we've successfully merged our resources at the Turing into the mybinder.org infrastructure!
+> This achievement marks Binder's maturity as a federated project and the Turing's commitment to supporting open-source tooling and communities.
+>
+> -- Dr Sarah Gibson
+
+![Federation](https://miro.medium.com/max/1666/1*KU35naJhl1LSDxKY8gog4g.png)
+
+## How Can You Get Involved?
+
+If this has piqued your interest, there are a few ways to get involved in the Binder Federation!
+
+One way is to become a [mybinder.org](https://mybinder.org) operator.
+This is a low-tech contribution which involves hanging out in the project's [gitter room](https://gitter.im/jupyterhub/binder) answering questions, providing constructive input in the [GitHub repo](https://github.com/jupyterhub/binderhub), and being an active member of the community.
```

Comment:
What do you want to say with "low-tech" here? It somehow makes me think it is "not a real contribution". Somehow  all the negative connotations of "not a code contribution" :-/ Can we formulate it as a positive? (Trying to think of something but drawing a blank right now)

Reply:
"This pathway does not require you to provide computational resources, but instead hang out in gitter..." ?

Reply:
Ja, dunno. Need to think before I can say something :-/

Reply:
I agree with @betatim's gut feeling here. What about: "There are no specific technical skills required for this contribution. It involves hanging out ...."

Reply:
How about "this is a job that requires skills such as working as a distributed team, planning ahead, empathy with our users, and not getting stressed when the service is down and we don't (yet) know why". Defining it via the things you do need or we would love to see more of instead of the things we think you might not need.

What is a good way to summarise "you derive pleasure from helping build and run a thing that most of the time for most of its users 'just works'. It doesn't stress you out that you have the power to shutdown a system that potentially several classes and workshops are using right this moment to teach people new things. Coordinating all this with other humans over the internet is fun for you."

Reply:
"This is a job for people who are passionate about building a slick service for a diverse range of users"?